.buildkite/scripts/steps/serverless/build_and_deploy.sh

```diff
@@ -137,8 +137,31 @@ EOF
   buildkite-agent meta-data set pr_comment:early_comment_job_id "$BUILDKITE_JOB_ID"
 }
 
+# This is the integration with the observability-test-environments
+create_github_issue_oblt_test_environments() {
+
+cat <<EOF > .issue-body
+### Kibana image
+
+$KIBANA_IMAGE
+
+### Further details
+
+Caused by $GITHUB_PR_TRIGGER_USER using the github label in $BUILDKITE_REPO/pull/$BUILDKITE_PULL_REQUEST
+EOF
+
+  gh issue create \
```

Comment:
Question, what env variable should be used? I see two different ones:
- https://github.com/elastic/kibana/blob/98354a2ae21fcba7f3d4bef791fea4999bb182fa/.buildkite/scripts/common/setup_job_env.sh#L16-L20

And https://github.com/elastic/kibana/blob/98354a2ae21fcba7f3d4bef791fea4999bb182fa/.buildkite/scripts/steps/build_api_docs.sh#L31

```suggestion
  GH_TOKEN="$KIBANA_CI_GITHUB_TOKEN" \
  gh issue create \
```

Reply:
You may need a specific user/token for `elastic/observability-test-environments` repo. Unsure if the bots have access to that repo.

Reply:
I will try using `KIBANA_CI_GITHUB_TOKEN` and grant write access to that particular repository.

Reply:
Doesn't matter much, as long as we aren't rate limited.  lgtm